cmd/eudico/mirvalidator/run.go

```diff
@@ -81,7 +81,14 @@ var runCmd = &cli.Command{
 		&cli.IntFlag{
 			Name:  "segment-length",
 			Usage: "The length of an ISS segment. Must not be negative",
-			Value: 1,
+		},
+		&cli.DurationFlag{
```

Comment:
This is another of those parameters like `segmentLength` that is dangerous to make them configurable as peers in the same subnet may end up setting different config parameters and leading to inconsistencies. We should maybe consider making it a compilation parameter instead of a flag to avoid this from happening, WDYT? 

@matejpavlovic, correct me if I am wrong, setting this value differently in different validators of the same subnet could be catastrophic, right? (an actually hard to debug)

Reply:
What if the users want to create a subnet with specific Mir parameters? I would prefer to use compilation parameters for the networks like Mainnet, but not for networks controlled by users by design.

Reply:
1) Yes, it would completely break the system, if different validators have different configurations.

2) I think both approaches (compile-time and start-time) have pros and cons. I suggest doing what is easier of you (time and effort invested seems to be the first priority right now), and think properly about this after launch. It's easy to change for the next upgrade.